formatTest/unit_tests/expected_output/variants.re

```diff
@@ -514,3 +514,25 @@ module Graph = {
 
 type Graph.node +=
   | Str = Graph.Str;
+
+type water = ..;
+
+type water +=
+  pri
+  | Ocean;
+
+type water +=
+  pri
```

Comment:
I agree. And the leading bar makes it clear that it applies too all not just the first constructor.